addOns/oast/oast.gradle.kts

```diff
@@ -0,0 +1,16 @@
+description = "OAST Support: Exploit Out-Of-Band Vulnerabilities"
+
+zapAddOn {
+    addOnName.set("OAST Support")
+    zapVersion.set("2.10.0")
+
+    manifest {
+        author.set("ZAP Dev Team")
+        url.set("https://www.zaproxy.org/docs/desktop/addons/oast-support/")
```

Comment:
Is this URL based on what's set here? Or is it based on build file properties?

Reply:
The later part is based on the main help TOC item (`OAST Support` which after normalisation is `oast-support`).

Reply:
Oh Okay. Thanks